<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BinaryTreeGenerator</class>
 <widget class="QMainWindow" name="BinaryTreeGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BinaryTreeGenerator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addNodeBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить узел</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addNodeLine">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>1590</width>
      <height>980</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="printTreeBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вывести базовое
дерево (поиска)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="printPBBTBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вывести PBBT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="printHeightsBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Высота</string>
    </property>
   </widget>
   <widget class="QPushButton" name="printMaxBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Максимальное
значение узла</string>
    </property>
   </widget>
   <widget class="QPushButton" name="printMinBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Минимальное
значение узла</string>
    </property>
   </widget>
   <widget class="QLabel" name="minValueLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Press
Btn</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="maxValueLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Press
Btn</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="heightsValueLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Base: x
PBBT: z</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
